create-custom-map-style/tileserver/downloadSpritesGlyphs.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+#!/bin/bash
+
+mkdir sprites
+mkdir glyphs
+cd sprites
+aws location get-map-sprites --file-name sprites.png --map-name $1 --profile $2 sprites.png
+aws location get-map-sprites --file-name sprites@2x.png --map-name $1 --profile $2 sprites@2x.png
+aws location get-map-sprites --file-name sprites.json --map-name $1 --profile $2 sprites.json
+aws location get-map-sprites --file-name sprites@2x.json --map-name $1 --profile $2 sprites@2x.json
+cd ../glyphs
+mkdir Ubuntu%20Medium%20Italic
```

Comment:
Is the `%20` (here, and below) on purpose? or is it some wacky html translation error

Reply:
Interesting question @awstalldave, this was originally part of an experiment to see if we could serve the sprites and glyphs from the local file system using http-server. When you make a request to Location Service you have to URL-encode the string, which in this case is "Ubuntu Medium Italic".